### DISCUSSION TITLE
Is it available in maven repo to be used as library ?

### DISCUSSION THREAD

**Discussion by PankajChandaTide** [sqids/sqids-java#8] [General]
I wanted to use it in my code, but i dont see it available in any remote repo, or am i doing any mistake ?

**Comment by 0x3333** [comment]
This week it will be available in the maven repo. Currently, there is a 0.1.0 snapshot version on:

https://s01.oss.sonatype.org/content/repositories/snapshots/org/sqids/sqids/0.1.0-SNAPSHOT/
